analyses/focal-cn-file-preparation/01-prepare-cn-file.R

```diff
@@ -102,18 +112,20 @@ process_annotate_overlaps <- function(cnv_df,
                                           keys = ensembl,
                                           column = "SYMBOL",
                                           keytype = "ENSEMBL"),
-      cytoband = AnnotationDbi::mapIds(org.Hs.eg.db::org.Hs.eg.db,
+      cytoband_org = AnnotationDbi::mapIds(org.Hs.eg.db::org.Hs.eg.db,
                                        keys = ensembl,
                                        column = "MAP",
                                        keytype = "ENSEMBL")
-    )
+    ) %>%
+    dplyr::select(-cytoband, -cytoband_org) %>%
```

Comment:
?

Reply:
I am removing the `cytoband` information here (as we discussed that it does not need to be included in the final table). Do you still agree? (In the upcoming commit, the `cytoband_org` variable will be removed altogether)